dockers/docker-ptf/Dockerfile.j2

```diff
@@ -40,7 +40,10 @@ RUN sed --in-place 's/httpredir.debian.org/debian-archive.trafficmanager.net/' /
         wget                \
         cmake               \
         libqt5core5a        \
-        libqt5network5
+        libqt5network5      \
+    && dpkg -r inetutils-ping \
+    && apt-get install -y iputils-ping
```

Comment:
Now that we're squashing dockers, this would be cleaner if broken out into a separate `RUN dpkg -r inetutils-ping` statement. Thoughts?